manifests/init.pp

```diff
@@ -163,6 +167,8 @@
   contain telegraf::config
   contain telegraf::service
 
+  create_resources(telegraf::processor, $processors)
```

Comment:
please don't use the legacy `create_resource()` function. Instead, iterate on $processors like we do it here: https://github.com/voxpupuli/puppet-ferm/blob/master/manifests/init.pp#L96-L100